### ISSUE TITLE
Chorale score URLs (Bach and Goudimel)

### ISSUE THREAD

**Issue by MarkGotham** [MarkGotham/When-in-Rome#93]
In flux.

Settle and update here.

**Comment by MarkGotham** [comment]
Provisionally
- [this for Bach](https://raw.githubusercontent.com/MarkGotham/Chorale-Corpus/Bach_JS)
- [and this for Goudimel](https://raw.githubusercontent.com/MarkGotham/Chorale-Corpus/Goudimel_C)

**Comment by MarkGotham** [comment]
When this is sorted then the only known missing file should be `op74n3-01.krn`.

So then ready for a new unittest.

```
self.assertEqual(
    test_remote_urls.main(),
    ["https://raw.githubusercontent.com/napulen/humdrum-haydn-quartets/master/kern/op74n3-01.krn"]
)

```

**Comment by manoskary** [comment]
> Provisionally
> 
>     * [this for Bach](https://raw.githubusercontent.com/MarkGotham/Chorale-Corpus/Bach_JS)
> 
>     * [and this for Goudimel](https://raw.githubusercontent.com/MarkGotham/Chorale-Corpus/Goudimel_C)


Greetings, 

thank you for all the great work. I noticed that some of the sources for the Goudimel chorales are missing.
Are there any updates?

Thank you in advance.

**Comment by MarkGotham** [comment]
Hey Emmanouil / Manos,

Thanks for this nudge and sorry for the slow reply.

You reminded me to get back on this, and doing so reminded me why I hadn't just done it in the first place ... it's a bit tricky!

Anyway, I've now solved this to my satisfaction ... at the expense of complicating matters ;) ... that should be good for flexible future uses.

As you'll see the source repository stores the data in `json` and produces ancient and modern versions of the scores. 

I've made the json, uploaded that with the code the code and the rendered ancient versions. After a bit more checking on that, I'll render the modern ones and close this issue.

LMK
- if you notice any issues.
- if you have any ideas for how to organise this data, beyond what I've done
- want me to duplicate the modern versions of the scores here (as they're from scratch, that's fine)

Another reason for the delay is that we're just in the process of making a much more epic, multi-group, multi-composer collection with the Bach, Goudimel, and many more. Watch this space!

Ok, essay over ;)
